<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIDCalibrationDialog</class>
 <widget class="QMainWindow" name="PIDCalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1495</width>
    <height>698</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Still Dashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background: white;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>23</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Column Top</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>600</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 138, 179)</string>
    </property>
    <property name="text">
     <string>Minutes since startup</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>23</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Boiler</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_47">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-width: 6px;
	border-style: solid none none none;
	border-color: rgb(0,138,179);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 138, 179)</string>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 138, 179)</string>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="Heat_Graph">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>951</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>23</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Column Middle</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="Temp_Graph">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>43</y>
      <width>951</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_43">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-width: 6px;
	border-style: solid none none none;
	border-color: rgb(157,224,234);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_48">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-width: 6px;
	border-style: solid none none none;
	border-color: rgb(0,44,119);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="Messages">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>461</width>
      <height>150</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Alarm</string>
     </property>
     <property name="background">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>203</red>
        <green>34</green>
        <blue>91</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Warning</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>203</red>
        <green>34</green>
        <blue>91</blue>
       </color>
      </brush>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Message</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sucess</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>7</red>
        <green>155</green>
        <blue>132</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 138, 179)</string>
    </property>
    <property name="text">
     <string>Messages</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>300</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 138, 179)</string>
    </property>
    <property name="text">
     <string>Minutes since startup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boiler_Temp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color:rgb(191,191,191);
	font: 12pt &quot;Arial&quot;;
	border: 2px solid black;
}

QPushButton:pressed {
    background-color: grey;
}</string>
    </property>
    <property name="text">
     <string>Current temp: XXXXXX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Column_1">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>10</y>
      <width>21</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color:rgb(191,191,191);;
	font: 12pt &quot;Arial&quot;;
	border: 2px solid black;
}

QPushButton:pressed {
    background-color: grey;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Column_2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>10</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color:rgb(191,191,191);;
	font: 12pt &quot;Arial&quot;;
	border: 2px solid black;
}

QPushButton:pressed {
    background-color: grey;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Column_3">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>30</y>
      <width>10</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color:rgb(191,191,191);;
	font: 12pt &quot;Arial&quot;;
	border: 2px solid black;
}

QPushButton:pressed {
    background-color: grey;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MidCol_Temp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color:white;
	font: 12pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string>Current temp: XXXXXX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TopCol_Temp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color:white;
	font: 12pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string>Current temp: XXXXXX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="master_Heat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color: rgb(226, 152, 21);
	font: 12pt &quot;Arial&quot;;
	color:white;
	border: 2px solid black;
}

QPushButton:pressed {
    background-color: grey;
}</string>
    </property>
    <property name="text">
     <string>Master heat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Heat_100">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set to 100%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Heat_50">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set to 50%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Heat_Other">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Other setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Heat_off">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heat off</string>
    </property>
   </widget>
   <widget class="QLabel" name="alarm_Text">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(203, 34, 91)</string>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 138, 179)</string>
    </property>
    <property name="text">
     <string>Alarm switch:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Turn_Off_Alarm">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Turn off alarm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boiler_Heat">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 0px;
    background-color: rgb(226, 152, 21);
	font: 12pt &quot;Arial&quot;;
	color:white;
	border: 2px solid black;
}

QPushButton:pressed {
    background-color: grey;
}</string>
    </property>
    <property name="text">
     <string>Boiler heat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1495</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
